src/schema/v2/me/myCollectionCollectedArtists.ts

```diff
@@ -0,0 +1,87 @@
+import { GraphQLBoolean, GraphQLFieldConfig, GraphQLInt } from "graphql"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { pageable } from "relay-cursor-paging"
+import { ResolverContext } from "types/graphql"
+import { artistConnection } from "../artist"
+import { paginationResolver } from "../fields/pagination"
+import ArtistSorts from "../sorts/artist_sorts"
+
+export const MyCollectionCollectedArtists: GraphQLFieldConfig<
+  any,
+  ResolverContext
+> = {
+  description: "A connection of artists in the users' collection",
+  type: artistConnection.connectionType,
+  args: pageable({
+    sort: ArtistSorts,
+    page: { type: GraphQLInt },
+    size: { type: GraphQLInt },
+    includePersonalArtists: {
+      type: GraphQLBoolean,
+      defaultValue: false,
+      description: "Include artists that have been created by the user.",
+    },
+  }),
+  resolve: async (
+    _root,
+    args,
+    { collectionArtistsLoader, meUserInterestsLoader, userID }
+  ) => {
+    if (!collectionArtistsLoader || !meUserInterestsLoader) return
+
+    const { page, offset, size, sort } = convertConnectionArgsToGravityArgs(
+      args
+    )
+
+    // TODO: Remove this once all clients pass includePersonalArtists correctly
+    // This is a hack we need to return the correct results for older cleints that don't send the param `includePersonalArtists`.
+    // With this solution we are defaulting to true if the query comes from the My Collection artwork form (which is the only query that passes `first: 100`)
```

Comment:
This was already in there. @MounirDhahri do you know if and when we can remove this hack?